recipes/pyflowy/recipe.yaml

```diff
@@ -0,0 +1,85 @@
+schema_version: 1
+
+context:
+  name: pyflowy
+  version: 0.0.1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/pyflowy-${{ version }}.tar.gz
+  sha256: babe8f58e1d2cc3784a87a50bfbb904cac088ad51259a7a2ccb835d5e3f0dfd1
+
+build:
+  skip:
+    - win
+    - osx
+  number: 0
+  script:
+    - ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib('c') }}
+    - cmake
+    - meson
+    - ninja
+    - xtensor <0.25.0
+    - xtensor-blas
+    - xtensor-python
+    - if: (build_platform != target_platform)
+      then:
+        - cross-python_${{ target_platform }}
+        - numpy
+        - pybind11 >=2.12.0
+        - python
+  host:
+    - fmt
+    - hdf5
+    - libcblas
+    - libnetcdf
+    - meson-python
+    - pip
+    - pkgconfig
+    - pybind11
+    - python
+    - numpy
+  run:
+    - fmt
+    - hdf5
+    - libnetcdf
```

Comment:
```suggestion
```
These libraries should already have run exports, so they shouldn't need to be explicitly added here.